crates/tx-graph/src/pog_musig_functor.rs

```diff
@@ -0,0 +1,339 @@
+use std::future::Future;
+
+use futures::future::join_all;
+use serde::{Deserialize, Serialize};
+use strata_bridge_primitives::constants::NUM_ASSERT_DATA_TX;
+
+use crate::transactions::assert_chain::{deserialize_assert_vector, serialize_assert_vector};
+
+const NUM_PAYOUT_OPTIMISTIC_INPUTS: usize = 5;
+const NUM_PAYOUT_INPUTS: usize = 4;
+const NUM_SLASH_STAKE_INPUTS: usize = 2;
```

Comment:
Why not also move these to `strata_bridge_primitives::constants` where `NUM_ASSERT_DATA_TX` is located.

Reply:
Idk I've gone back and forth on this. These constants aren't really constants as much as they are derived properties of the PegOutGraph construction that we happen to know up front. Unlikely the NUM_ASSERT_DATA_TX which I think is a more fundamental parameter. I could be wrong but they seem different to me. As far as I can tell, this is really the only scope in which they've been needed but I haven't scoured the whole codebase to confirm that.

Reply:
Then we should move them to `Params`, i.e. stuff that gets setup on genesis and won't change.
Anyways, resolving this for now. I am happy with them being constants somewhere.